From f0e521daf0a77826a47ef9e80b8216b46401073f Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 1 Jun 2018 16:10:48 +0800
Subject: [PATCH 3895/5242] MLK-18477-1 arm64: fsl-imx8qm.dtsi: Add bypass and
 disp_sel clks for dpu

commit  0beb0c1b1f7fbe0f9140f7408e6544dfcadde703 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds bypass clocks and disp_sel clocks in the dpu nodes.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |   12 ++++++++++--
 1 file changed, 10 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 44dfffa..9b56b51 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -1669,9 +1669,13 @@
 				  "irq_dpr1";
 		clocks = <&clk IMX8QM_DC0_PLL0_CLK>,
 			 <&clk IMX8QM_DC0_PLL1_CLK>,
+			 <&clk IMX8QM_DC0_BYPASS_0_DIV>,
+			 <&clk IMX8QM_DC0_DISP0_SEL>,
+			 <&clk IMX8QM_DC0_DISP1_SEL>,
 			 <&clk IMX8QM_DC0_DISP0_CLK>,
 			 <&clk IMX8QM_DC0_DISP1_CLK>;
-		clock-names = "pll0", "pll1", "disp0", "disp1";
+		clock-names = "pll0", "pll1", "bypass0",
+			      "disp0_sel", "disp1_sel", "disp0", "disp1";
 		assigned-clocks = <&clk IMX8QM_DC0_DISP0_SEL>,
 				  <&clk IMX8QM_DC0_DISP1_SEL>;
 		assigned-clock-parents = <&clk IMX8QM_DC0_PLL0_CLK>,
@@ -2133,9 +2137,13 @@
 				  "irq_dpr1";
 		clocks = <&clk IMX8QM_DC1_PLL0_CLK>,
 			 <&clk IMX8QM_DC1_PLL1_CLK>,
+			 <&clk IMX8QM_DC1_BYPASS_0_DIV>,
+			 <&clk IMX8QM_DC1_DISP0_SEL>,
+			 <&clk IMX8QM_DC1_DISP1_SEL>,
 			 <&clk IMX8QM_DC1_DISP0_CLK>,
 			 <&clk IMX8QM_DC1_DISP1_CLK>;
-		clock-names = "pll0", "pll1", "disp0", "disp1";
+		clock-names = "pll0", "pll1", "bypass0",
+			      "disp0_sel", "disp1_sel", "disp0", "disp1";
 		assigned-clocks = <&clk IMX8QM_DC1_DISP0_SEL>,
 				  <&clk IMX8QM_DC1_DISP1_SEL>;
 		assigned-clock-parents = <&clk IMX8QM_DC1_PLL0_CLK>,
-- 
1.7.9.5

